tests/functional/rbac/test_reassignment.py

```diff
@@ -149,6 +154,137 @@ def grant_role(self, client: ADCMClient, user: User, role: RbacRoles, *objects:
                 name=f'{user.username} is {role.value}', role=client.role(name=role.value), objects=objects, user=[user]
             )
 
+    def test_add_remove_user_from_group_and_policy(self, clients, is_denied_to_user, prepare_objects, user):
+        """Test that policies are applied after user add/remove to the policy"""
+
+        admin_cluster, admin_service, admin_component, *_ = prepare_objects
+
+        is_denied_to_user(admin_cluster, BusinessRoles.EditClusterConfigurations)
+        is_denied_to_user(admin_service, BusinessRoles.EditServiceConfigurations)
+        is_denied_to_user(admin_component, BusinessRoles.EditComponentConfigurations)
+        with allure.step('Create user group'):
```

Comment:
Maybe 'create a group with user "test_user"'?

Reply:
what the difference?

Reply:
more information on what's in the group

Reply:
changed